app/config/AppConfig.scala

```diff
@@ -114,9 +121,11 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, configuration: Configu
 
   def userAllowListBaseUrl: String = s"$userAllowListService/user-allow-list"
 
-  private def referenceDataService: String = servicesConfig.baseUrl("reference-data")
+  private def referenceDataService: String = servicesConfig.baseUrl("emcs-tfe-reference-data")
```

Comment:
[Optional] can this be a lazy val instead

Reply:
As this change has gone in across other repos, I'd prefer to keep it as is